service/src/test/java/bio/terra/cbas/controllers/VerifyPactsRunSetsApiController.java

```diff
@@ -184,4 +188,48 @@ public void runSetsData() throws Exception {
     when(smartRunSetsPoller.updateRunSets(response))
         .thenReturn(new TimeLimitedUpdater.UpdateResult<>(response, 1, 1, true));
   }
+
+  @State({"a run set with UUID 20000000-0000-0000-0000-000000000002 exists"})
```

Comment:
we had talked about consolidating the run and run set setup into a single state function, but actually, I think that is not as straightforward as I thought it would be -- so I think we can leave figuring that out for another time.